pager/lib.h

```diff
@@ -81,4 +81,6 @@ void mutt_buffer_strip_formatting(struct Buffer *dest, const char *src, bool str
 
 void mutt_clear_pager_position(void);
 
+bool config_init_pager(struct ConfigSet *cs);
```

Comment:
Actually, this isn't needed.
The macro in `mutt_config.c` declares the function for you:
https://github.com/neomutt/neomutt/blob/8ff8cd7f5e5f773c257522218583e9d36324c237/mutt_config.c#L71-L73

Reply:
I was mimicking `index/lib.h`

Reply:
Hmm... a quick grep shows 17 `config_init_*()` functions.
We don't need any of them.  Please can you delete them.

Reply:
I removed declarations. Do we need to remove definitions too?